packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -290,6 +294,25 @@ export function applyUndoRedoEdit(
 	}
 }
 
+export function applyConstraint(state: FuzzTestState, constraint: Constraint) {
+	const tree = viewFromState(state);
+	switch (constraint.content.type) {
+		case "nodeConstraint": {
+			const constraintNodePath = constraint.content.content;
+			const constraintNode =
+				constraintNodePath !== undefined
+					? navigateToNode(tree, constraintNodePath)
+					: undefined;
+			if (constraintNode !== undefined) {
+				tree.checkout.editor.addNodeExistsConstraint(constraintNode.anchorNode);
+			}
+			break;
+		}
+		default:
+			break;
```

Comment:
Can we put an `unreachableCase` here, or at least a runtime assertion is we encounter an unhandled constraint type?

Reply:
Added `unreachableCase` to default case